sdk/identity/azure-identity/tests/test_authn_client.py

```diff
@@ -15,7 +15,8 @@
 from azure.core.credentials import AccessToken
 from azure.identity._authn_client import AuthnClient
 from six.moves.urllib_parse import urlparse
-from helpers import mock_response
+
+from helpers import mock_response, Request
```

Comment:
I looked through the file, not seeing where `Request` is used, am I missing it?

Reply:
No, that's an unused import, thanks for spotting it.